src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,13 @@
+package wolox.training.repositories;
+
+import java.util.Collection;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+import wolox.training.models.Book;
+
+public interface BookRepository extends CrudRepository<Book, Long> {
+  @Query("SELECT book from Book book WHERE book.author = :author")
```

Comment:
Implementing one of the Repository interfaces, the DAO will already have some basic CRUD methods (and queries) defined and implemented. When Spring Data creates a new Repository implementation, it analyses all the methods defined by the interfaces and tries to automatically generate queries from the method names. 

[Spring data JPA](https://www.baeldung.com/the-persistence-layer-with-spring-data-jpa)